.github/ISSUE_TEMPLATE/idea_proposal.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Idea proposal
+about: Propose a new idea for discussion to improve the Dataverse software!
+title: 'Suggestion:'
+labels: 'Type: Suggestion'
+assignees: ''
+
+---
+
+<!--
+Thank you for contributing to the Dataverse Project through the creation of a feature request!
+
+More information on ideas/feature requests and contributions can be found in the "Contributing to Dataverse" page:
+https://github.com/IQSS/dataverse/blob/develop/CONTRIBUTING.md#ideasfeature-requests
```

Comment:
Heads up that in this PR we plan to move content from CONTRIBUTING.md to its own guide:

- #10532

So that #ideasfeature-requests link won't work anymore.